__tests__/app.test.js

```diff
@@ -49,4 +51,56 @@ describe("GET /api/topics", () => {
             expect(res.body.topics.length).toBe(3);
         });
     });
+})
+
+describe("GET /api/articles/:article_id", () => {
+    test("200: Get the correct status code", () => {
+        return request(app).get("/api/articles/1").expect(200);
```

Comment:
Don't really need the first two tests with server testing - we check the status and that the request is sending back an array of objects implicitly in the 200 test on line 71 which is fine